tests/frontier/opcodes/test_all_opcodes.py

```diff
@@ -0,0 +1,106 @@
+"""
+Call every possible opcode to see if it is recognized by the evm
+If opcode is undefined the subcall will fail
```

Comment:
```suggestion
Call every possible opcode and test that the subcall is successful
if the opcode is supported by the fork supports and fails otherwise.
```